### PR TITLE
Add Euro Truck Simulator 2

### PR DESCRIPTION
**Pull request by IceOfWraith**
This adds Euro Truck Simulator 2 which is a lift and shift of American Truck Simulator. I believe I caught all references and updated them. I also performed testing using a copy of the game and was able to connect and play the same.

### CODE REVIEW

File: euro-truck-simulator-2.kvp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+App.MonitorChildProcess=False
+App.DumpFullChildProcessTree=False
+App.MonitorChildProcessWaitMs=1000
+App.MonitorChildProcessName=
+App.SupportsUniversalSleep=False
+App.WakeupMode=Any
+App.ApplicationReadyMode=Immediate
+App.AttemptedExtIPFetch=True
+App.IPFetchLimiter=System.Threading.SemaphoreSlim
+App.LastFailedIPFetch=null
```
Comment (Greelan) [inline euro-truck-simulator-2.kvp:79]: These 3 lines can be deleted.

File: euro-truck-simulator-2config.json [hunk 2]
```diff
@@ -0,0 +1,340 @@
+[
+    {
+        "DisplayName":"Lobby Name",
+        "Category":"ETS - Server Settings",
+        "Description":"Session name, limited to 63",
```
Comment (Greelan) [inline euro-truck-simulator-2config.json:5]: Missing "characters" at the end.

**Review by Greelan — CHANGES_REQUESTED** [review]
Desktop review (like ATS, I don't own the game). Couple of nits only.

File: euro-truck-simulator-2config.json (hunk 2 above)
Comment (IceOfWraith) [reply euro-truck-simulator-2config.json]: Went ahead and fixed this in ATS on this PR as well. Hope you don't mind.

**Review by Greelan — APPROVED** [review]

**Pull request merged**